api/conf/schema.json

```diff
@@ -1 +1 @@
:"http://127.0.0.1:10080"},"appid":{"type":"string","default":"unset"},"header_prefix":{"type":"string","default":"X-"}}}},"log-rotate":{"schema":{"type":"object","additionalProperties":false,"properties":[]}},"response-rewrite":{"schema":{"type":"object","minProperties":1,"additionalProperties":false,"properties":{"body_base64":{"description":"whether new body for repsonse need base64 decode before return","type":"boolean","default":false},"status_code":{"description":"new status code for repsonse","maximum":598,"type":"integer","minimum":200},"body":{"description":"new body for repsonse","type":"string"},"headers":{"description":"new headers for repsonse","minProperties":1,"type":"object"}}}},"consumer-restriction":{"schema":{"type":"object","oneOf":[{"required":["blacklist"],"title":"blacklist","properties":{"type":{"type":"string","enum":["consumer_name","service_id"],"default":"consumer_name"},"rejected_code":{"type":"integer","default":403,"minimum":200},"blacklist":{"type":"array","items":{"type":"string"},"minItems":1}}},{"required":["whitelist"],"title":"whitelist","properties":{"type":{"type":"string","enum":["consumer_name","service_id"],"default":"consumer_name"},"rejected_code":{"type":"integer","default":403,"minimum":200},"whitelist":{"type":"array","items":{"type":"string"},"minItems":1}}}]}},"redirect":{"schema":{"type":"object","oneOf":[{"required":["uri"]},{"required":["http_to_https"]}],"properties":{"http_to_https":{"type":"boolean"},"ret_code":{"type":"integer","default":302,"minimum":200},"uri":{"type":"string","minLength":2}}}},"proxy-rewrite":{"schema":{"type":"object","minProperties":1,"additionalProperties":false,"properties":{"scheme":{"description":"new scheme for upstream","enum":["http","https"],"type":"string"},"host":{"description":"new host for upstream","pattern":"^[0-9a-zA-Z-.]+$","type":"string"},"regex_uri":{"description":"new uri that substitute from client uri for upstream, lower priority than uri property","type":"array","minItems":2,"items":{"description":"regex uri","type":"string"},"maxItems":2},"headers":{"description":"new headers for request","minProperties":1,"type":"object"},"uri":{"description":"new uri for upstream","maxLength":4096,"type":"string","pattern":"^\\/.*","minLength":1}}}},"batch-requests":{"schema":{"type":"object","additionalProperties":false}},"prometheus":{"schema":{"type":"object","additionalProperties":false}},"echo":{"schema":{"anyOf":[{"required":["before_body"]},{"required":["body"]},{"required":["after_body"]}],"properties":{"after_body":{"description":"body after the modification of filter phase.","type":"string"},"headers":{"description":"new headers for response","minProperties":1,"type":"object"},"before_body":{"description":"body before the filter phase.","type":"string"},"auth_value":{"description":"auth value","type":"string"},"body":{"description":"body to replace upstream response.","type":"string"}},"type":"object","minProperties":1,"additionalProperties":false}}}}
```

Comment:
it is hard to confirm the new result is the correct one.

I think we can use YAML or formatted JSON style. JSON or YAML, the results need to be in order, which makes it easier to compare changes.

Anyway, I'll create a new issue for this, we can implement this feature in a new PR.

```json
{
	"traceSegmentId": "4560dd63-1030-4905-88e1-8496223f1925",
	"traceId": "4560dd63-1030-4905-88e1-8496223f1925",
	"service": "APISIX",
	"spans": [{
		"peer": "upstream service",
		"spanId": 1,
		"spanLayer": "Http",
		"spanType": "Exit",
		"startTime": 1603598158143,
		"endTime": 1603598158143,
		"operationName": "/hello",
		"componentId": 6000,
		"isError": false,
		"tags": [{
			"key": "http.status",
			"value": 200
		}],
		"parentSpanId": 0
	}, {
		"spanId": 0,
		"spanLayer": "Http",
		"spanType": "Entry",
		"startTime": 1603598158143,
		"endTime": 1603598158143,
		"operationName": "/hello",
		"componentId": 6000,
		"isError": false,
		"tags": [{
			"key": "http.method",
			"value": "GET"
		}, {
			"key": "http.params",
			"value": "http://127.0.0.1/hello"
		}, {
			"key": "http.status",
			"value": "200"
		}],
		"parentSpanId": -1
	}],
	"serviceInstance": "APISIX Instance Name"
}
```

```yaml
traceSegmentId: 4560dd63-1030-4905-88e1-8496223f1925
traceId: 4560dd63-1030-4905-88e1-8496223f1925
service: APISIX
spans:
  - peer: upstream service
    spanId: 1
    spanLayer: Http
    spanType: Exit
    startTime: 1603598158143
    endTime: 1603598158143
    operationName: /hello
    componentId: 6000
    isError: false
    tags:
      - key: http.status
        value: 200
    parentSpanId: 0
  - spanId: 0
    spanLayer: Http
    spanType: Entry
    startTime: 1603598158143
    endTime: 1603598158143
    operationName: /hello
    componentId: 6000
    isError: false
    tags:
      - key: http.method
        value: GET
      - key: http.params
        value: 'http://127.0.0.1/hello'
      - key: http.status
        value: '200'
    parentSpanId: -1
serviceInstance: APISIX Instance Name
```

Reply:
new issue: https://github.com/apache/apisix-dashboard/issues/672

let us discuss in the issue.

Reply:
got it~!